test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,73 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+
+    /**
+     * Test the most obvious case
+     */
+    @Test 
+    public void isSimilar_totallySimilar() throws IllegalValueException {
+	Name name1 = new Name("John Smith");
```

Comment:
going further you can consider all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.